src/utils/IdealGraphVisualizer/Data/src/main/java/com/sun/hotspot/igv/data/InputGraph.java

```diff
@@ -39,10 +39,14 @@ public class InputGraph extends Properties.Entity implements FolderElement {
     private List<InputBlockEdge> blockEdges;
     private Map<Integer, InputBlock> nodeToBlock;
     private boolean isDiffGraph;
+    private InputGraph firstGraph, secondGraph;
```

Comment:
I split it into two lines. They can not be final because the constructor `InputGraph(InputGraph firstGraph, InputGraph secondGraph)` calls the second constructor `InputGraph(String name)` and both assign `firstGraph` and  `secondGraph`